app/controllers/users_controller.rb

```diff
@@ -1,5 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
```

Comment:
- [ ] It will be a good idea to add sample data to the `seed.rb` file so as to render sample posts on the fly.